docs/guides/custom-images.rst

```diff
@@ -0,0 +1,68 @@
+Using Custom Docker Images on Compute Engine
+=====================
+
+When running workflows remotely on Compute Engine, your tasks run in a container that uses one of the
+``orquestra-sdk-base`` images depending on whether you have requested a GPU or not (you can see their source code
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/Dockerfile>`_ and
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/cuda.Dockerfile>`_).
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Do you think we should mention that these images are automatically selected based on the version of the SDK your workflow was created with?

Reply:
Yeah I think that might be helpful.